cmd/maya-apiserver/app/server/snapshot_endpoint.go

```diff
@@ -167,6 +169,17 @@ func (s *HTTPServer) snapshotList(resp http.ResponseWriter, req *http.Request, v
 
 }
 
+// SnapshotClone is http handler for restore a snapshot to a persistent volume
```

Comment:
Some more explanation might be nice. Some questions that arise here are:
- Are the PV already available?
- Are the snapshots already available?
- Is this possible if all are in the same namespace, different namespaces, different clusters?

You need not put these info here. But this should be answered somewhere.
Was there a design doc that tries to talk about these things?

Reply:
Workflow is changed now, API no longer exists.